cli/core/hack/rpm/tanzu-cli.spec

```
@@ -0,0 +1,46 @@
+Name:       tanzu-cli
+Version:    %{cli_version}
+Release:    0
```

```suggestion
Release:    1
```
release should init with 1 and incremented with any changes in SPEC file for same minor

---

In the build script I actually get rid of this extra id.  But I see your point it could prove useful as you mention (hopefully not often 😄 ).  I will modify the PR to keep this id and start it at `1`.

---

@navidshaikh Say we publish a version 2 after changing the SPEC file, should we keep the version 1 package published as well or should we completely remove it and replace it with version 2?

---

ideally we keep both, yum/dnf will take care of picking latest if requested to install afresh / upgrade